src/lib/swapper/swappers/ThorchainSwapper/utils/getLongtailQuote.ts

```diff
@@ -32,22 +32,24 @@ export const getLongtailToL1Quote = async (
   streamingInterval: number,
   assetsById: AssetsByIdPartial,
 ): Promise<Result<ThorTradeQuote[], SwapErrorRight>> => {
+  const { sellAsset, sellAmountIncludingProtocolFeesCryptoBaseUnit } = input
+
   /*
     We only support ETH -> L1 for now.
```

Comment:
Very nitpicky but can we make this `Ethereum` vs. `ETH`, and add `long-tail`, `longtail` or `long tail` somewhere in this comment as well for grep purposes when this will need to be updated?

Reply:
https://github.com/shapeshift/web/pull/6196/commits/6faa8486e2dd1642d98687d68061437fda4dfd59